conf/systemd.service

```diff
@@ -1,13 +1,17 @@
 [Unit]
-Description=Free and open source comics/mangas media server.
+Description=Komga server
 After=network.target
 
 [Service]
 Type=simple
 User=__APP__
 Group=__APP__
 WorkingDirectory=__FINALPATH__/
-ExecStart=java -jar __FINALPATH__/app.jar --server.port=__PORT__ --server.servlet.context-path="__PATH__" >> /var/log/__APP__/__APP__.log 2>&1
+ExecStart=/usr/bin/java -jar -Xmx1g app.jar --server.port=__PORT__ --server.servlet.context-path="__PATH__/"
+Restart=on-failure
+RestartSec=10
+StandardOutput=null
+StandardError=syslog
```

Comment:
@ericgaspar Uuuh, why?

Reply:
@kay0u I followed the config proposed in this thread ... https://github.com/gotson/komga/issues/353

Reply:
Ok, the `line_match` argument can't work if we discard all logs. I fixed it here: https://github.com/YunoHost-Apps/komga_ynh/pull/6